pkgs/servers/matrix-synapse/matrix-appservice-irc/update.sh

```diff
@@ -15,10 +15,16 @@ fi
 
 echo "matrix-appservice-irc: $CURRENT_VERSION -> $TARGET_VERSION"
 
-sed -i "s/#$CURRENT_VERSION/#$TARGET_VERSION/" package.json
+# sed -i "s/#$CURRENT_VERSION/#$TARGET_VERSION/" package.json
+rm -f package.json package-lock.json
+wget https://github.com/matrix-org/matrix-appservice-irc/raw/$TARGET_VERSION/package.json
+wget -O package-lock-temp.json https://github.com/matrix-org/matrix-appservice-irc/raw/$TARGET_VERSION/package-lock.json
+echo -n "$TARGET_VERSION" > ./REVISION
```

Comment:
Just give it a newline and trim / get the first line in the nix revision. That makes this part more robust to manual updates & gets rid of the editorconfig error.